GeoBot/GeoBot/wwwroot/index.html

```diff
@@ -0,0 +1,87 @@
+<html>
+
+<body>
+    <div id="webchat" role="main"></div>
+    <script src="https://cdn.botframework.com/botframework-webchat/latest/webchat.js"></script>
+    <script>
+        function getUrlParam(name) {
+            var results = new RegExp('[\\?&]' + name + '=([^&#]*)').exec(window.location.href);
+            return (results && results[1]) || undefined;
+        }
+
+        var botname = getUrlParam('botname');
+        var endpoint = getUrlParam('endpoint');
+        
+        if (botname === undefined && endpoint === undefined) {
+            // if botname and endpoint is not set - we will use document.URL value as a default
+            hostname = document.location.hostname;
+            splitedHostname = hostname.split('.'); 
+            endpoint = "https://" + hostname;  
+            botname = splitedHostname[0];  
+        } 
+
+        function createFetchSpeechServicesCredentials(){
+            let expireAfter = 0;
+            let lastPromise;
+
+            return () => {
+                if (Date.now() > expireAfter) {
+                    const speechServiceUri = endpoint + "/speech/token";
```

Comment:
Isn't it possible to skip the endpoint? I would guess calls without endpoint will go to the same origin endpoint which will be the custom domain name or traffic manager url? [background](https://en.wikipedia.org/wiki/Ajax_%28programming%29)

```suggestion
                    const speechServiceUri = "/speech/token";
```

Reply:
Oh I didn't know that! Thank you for the suggestion! I will try & test. ;)